devito/stencilkernel.py

```diff
@@ -65,6 +65,7 @@ class StencilKernel(Function):
     """
     def __init__(self, stencils, **kwargs):
         name = kwargs.get("name", "Kernel")
+        self.name = name
```

Comment:
Could just do this in the above line and update all occurrences of `name` with `self.name` in the rest of the function.